src/main/kotlin/no/nav/familie/tilbake/behandling/BehandlingService.kt

```diff
@@ -46,7 +48,8 @@ class BehandlingService(private val behandlingRepository: BehandlingRepository,
                         private val behandlingskontrollService: BehandlingskontrollService,
                         private val stegService: StegService,
                         private val oppgaveTaskService: OppgaveTaskService,
-                        private val rolleConfig: RolleConfig) {
+                        private val rolleConfig: RolleConfig,
+                        private val dokumentbehandlingService: DokumentbehandlingService) {
```

Comment:
Du trenger ikke å kalle dokumentbehandlingService herfra. Du kan direkte opprette SendVarselBrevTask herfra. (via taskRepository) Behandlingen setter på vent automatisk

Reply:
Synes det er fint å ha en felles måte å sette på vent på slik at det gjøres likt for automatisk og manuellt sendt varsel.

Hvis jeg bare lager en task, vil sett på vent funk. ligge 2 forskjellige steder.

Reply:
stegService.fortsettBehandling som kalles etter behandling opprettelse setter behandling på vent automatisk. Derfor trenger du ikke å sette på vent.

Reply:
Jeg kommer til å endre behandlingskontroll senere slik at det brukes samme påvent funskjonalitet i hele familie-tilbake på  samme måte.